climada/engine/impact_forecast.py

```diff
@@ -88,3 +89,16 @@ def from_impact(
                 imp_mat=impact.imp_mat,
                 haz_type=impact.haz_type,
             )
+
+    def _check_sizes(self):
+        """Check sizes of forecast data vs. impact data.
+
+        Raises
+        ------
+        ValueError
+            If the sizes of the forecast data do not match the
+            :py:attr:`~climada.engine.impact.Impact.event_id`
+        """
+        num_entries = len(self.event_id)
+        size(exp_len=num_entries, var=self.member, var_name="Forecast.member")
```

Comment:
I would suggest to use a more explicit variable name in size (I know, not this PR, but I do not see the PR for size). 

Instead of `exp_len`, which can be confused within climada with `exposure_length`, maybe use `expected_length`.

Reply:
Correct, I reused it. But I agree 😁